pkg/didcomm/protocol/exchange/exchange.go

```diff
@@ -68,6 +68,11 @@ type Connection struct {
 	DIDDoc *did.Doc `json:"did_doc,omitempty"`
 }
 
+// provider contains dependencies for the Exchange protocol and is typically created by using aries.Context()
+type provider interface {
+	transport.OutboundTransport
```

Comment:
This should be a named field. We do not want to worry about name conflicts underneath.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/a26c159c381cbbb380ef97f616c8f9b9bddd476a..2002cc47490c6dea1ede21a12a1f5ebee834f700#diff-4d3ca323f9b237cfa9940c12efa9b615R73).